tiledb/sm/config/config.h

```diff
@@ -91,6 +91,16 @@ class Config {
    */
   static const std::string CONFIG_LOGGING_LEVEL;
 
+  /** An array will open between this value and timestamp_end upon a
+   * read query.
+   * */
+  static const std::string SM_ARRAY_OPEN_TIMESTAMP_START;
```

Comment:
See my renaming comment on dropping `open`. We should drop `OPEN` from these constants as well.